packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [25.3.0]
+
+### Added
+
+- Adds `tokenMethodIncreaseAllowance` transaction type ([#4069](https://github.com/MetaMask/core/pull/4069))
```

Comment:
When this is merged with `main`, there is an additional fix to the `TransactionController` with a suitable changelog entry on the PR, thanks!

https://github.com/MetaMask/core/pull/4101